Content.Server/Cargo/Components/CargoPalletConsoleComponent.cs

```diff
@@ -1,9 +1,18 @@
 using Content.Server.Cargo.Systems;
+using Content.Shared.Cargo.Prototypes;
 using Content.Shared.Stacks;
+using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 
 namespace Content.Server.Cargo.Components;
 
 [RegisterComponent]
 [Access(typeof(CargoSystem))]
-public sealed partial class CargoPalletConsoleComponent : Component;
+public sealed partial class CargoPalletConsoleComponent : Component
+{
+    /// <summary>
+    /// Moffstation - Makes a pallet send funds exclusively to one account
+    /// </summary>
+    [DataField("exclusiveAccount")]
```

Comment:
```suggestion
    [DataField]
```
Field name is identical to the implied one.